x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/external/request/huggingface/HuggingFaceElserRequestTests.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.inference.external.request.huggingface;
+
+import org.apache.http.HttpHeaders;
+import org.apache.http.client.methods.HttpPost;
+import org.elasticsearch.common.settings.SecureString;
+import org.elasticsearch.test.ESTestCase;
+import org.elasticsearch.xcontent.XContentType;
+import org.elasticsearch.xpack.inference.external.huggingface.HuggingFaceAccount;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import static org.elasticsearch.xpack.inference.external.http.Utils.entityAsMap;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+
+public class HuggingFaceElserRequestTests extends ESTestCase {
+    public void testCreateRequest() throws URISyntaxException, IOException {
+        var huggingFaceRequest = createRequest("www.google.com", "secret", "abc");
+        var httpRequest = huggingFaceRequest.createRequest();
+        httpRequest.getAllHeaders();
```

Comment:
The headers a tested below this line looks like it is left during refactoring and can be deleted?

Reply:
Oops, yeah copy/paste 🤦‍♂️